### ISSUE TITLE
Can't create custom getters and properties

### ISSUE THREAD

**Issue by Haidar0096** [rrousselGit/freezed#159]
I searched alot before I create this issue but couldn't find an answer.

This is my code:

```
@freezed
abstract class Board implements _$Board {
  const Board._();

  const factory Board() = _Board;

  int get x =>0;//I was defining a 2 -dimensional List here but I put an int instead just for demonstration 
}
```

and this is the error I am getting:

`@freezed cannot be used on classes with unimplemented getters`

Am I doing something wrong or is this not supported by freezed? I just followed the documentation [here](https://pub.dev/packages/freezed#custom-getters-and-methods
)  

Please help

**Comment by mhmdanas** [comment]
This issue was fixed in 0.17.0 (3a9fcfeb2565f55a9c00667d53d07fa52ac36fbf) and I can't reproduce it on that version. What version are you using?

**Comment by Haidar0096** [comment]
flutter version:
```
Flutter 1.12.13+hotfix.9 • channel stable •
https://github.com/flutter/flutter.git
Framework • revision f139b11009 (4 weeks ago) • 2020-03-30 13:57:30 -0700
Engine • revision af51afceb8
Tools • Dart 2.7.2

```

package version:
`freezed: ^0.10.4`

**Comment by Haidar0096** [comment]
> This issue was fixed in 0.17.0 ([3a9fcfe](https://github.com/rrousselGit/freezed/commit/3a9fcfeb2565f55a9c00667d53d07fa52ac36fbf)) and I can't reproduce it on that version. What version are you using?

I upgraded now to `^0.10.7` and the issue is fixed,thanks.

I will keep in mind for next time to check latest version  before posting an issue :)
